pkg/cli/cmd/radinit/recipe.go

```diff
@@ -73,7 +73,7 @@ func (drc *devRecipeClient) GetDevRecipes(ctx context.Context) (map[string]map[s
 	recipes := map[string]map[string]corerp.RecipePropertiesClassification{}
 
 	// if repository has the correct path it should look like: <registryPath>/recipes/<category>/<type>:<tag>
-	// Ex: radius.azurecr.io/recipes/local-dev/rediscaches:0.20
+	// Ex: ghcr.io/radius-project/recipes/local-dev/rediscaches:0.20
```

Comment:
Our prod registry is `ghcr.io/radius-project` and dev is `ghcr.io/radius-project/dev`, right? This logic checks everything under the `DevRecipeRegistry` (which is defined above) and goes through all the repos underneath that registry. If under `ghcr.io/radius-project/dev` we have recipes, they might get installed in the env too. The logic here should be very well tested with the new registry.

Another thing is that the logic here takes `radius-rp` as the starting point (see line 78). Was that repo also moved to **ghcr**? Were everything from **azurecr** moved to **ghcr**?

Reply:
yetkin and I updated this logic. should be good now